stories/VersionChooser.stories.tsx

```diff
@@ -8,7 +8,14 @@ import { fakeVersionsList } from '../src/test-helpers';
 import { renderWithStoreAndRouter } from './utils';
 
 const render = ({ addonId = 124, store = configureStore() } = {}) => {
-  return renderWithStoreAndRouter(<VersionChooser addonId={addonId} />, store);
+  const match = {
```

Comment:
Could we just dispatch a real `push` action similar to how we [did it on the frontend](https://github.com/mozilla/addons-frontend/blob/0c5b2ecb3440303bcece1919305b07cf82ba391f/tests/unit/amo/sagas/test_collections.js#L617-L620)?

I don't know if we're using a Redux-aware router yet, though.

Reply:
Ah, that could be an idea but we don't have a redux-aware router and I don't know if that'll be easy (see #48).

Reply:
I fixed this issue by using a `Route`.